source/fab/dep_tree.py

```diff
@@ -61,11 +64,13 @@ def __init__(self, fpath: Union[str, Path], file_hash: int, module_defs=None, sy
 
         assert all([d and len(d) for d in self.module_defs]), "bad module definitions"
         assert all([d and len(d) for d in self.symbol_defs]), "bad symbol definitions"
+        assert all([d and len(d) for d in self.module_deps]), "bad symbol dependencies"
```

Comment:
Is it a symbol dependency of a module dependency?

Reply:
typo, thanks (although it's technically both, as the module deps / defs are subsets of the symbol ones!)